strapi/strapi
# chore: bump strapi/eslint-config to 0.2.0

### What does it do?

* bumps `@strapi/eslint-config` to 0.2.0 and adds the additional dependencies required with it

### Why is it needed?

* support `front/typescript` configuration

### How to test it?

* lint tasks should pass
* You can also edit a file in your IDE to cause an eslint error e.g. remove the line break after any react import and it should error.

### Related issue(s)/PR(s)

* unblocks #17460
